go/libraries/doltcore/doltdb/root_val_test.go

```diff
@@ -98,3 +100,222 @@ func TestTableDiff(t *testing.T) {
 		t.Error("Bad table diff after adding a second table")
 	}
 }
+
+func TestDocDiff(t *testing.T) {
```

Comment:
ctx := context.Background()

one time at the top and then you can reuse it.  Save yourself a lot of typing.

Reply:
When a test gets long it can be helpful to sparsely populate it with comments.  Usually in a test like this you are mutating state, and then validating.  Before you mutate state saying what you are doing and what you will test can make long tests easier to read.

Reply:
Thanks for the feedback! Declared ctx at the top, and added comments